### ISSUE TITLE
G3 - Local Search has 2 search buttons (for screen-readers)

### ISSUE THREAD

**Issue by mmafe** [brharp/hjckrrh#116]
Two search buttons are available via screen-reader.

One of them says "button" the other says "search button."

Please adjust so second (redundant) search button cannot be accessed via screen-reader. Also please ensure that the remaining button says "search button."

**Comment by jszasz** [comment]
Issue #73 has some images for reference

**Comment by brharp** [comment]
Added descriptive text to first button, and hid second button from screen readers.
